### PR TITLE
Add inner hib bank and raven's point to ;banks.

### PR DESCRIPTION
**Pull request by Sarvatt**

### CODE REVIEW

**Comment by Sarvatt** [comment]
This is wrong, fixing it.

File: drinfomon.lic [hunk 1]
```diff
@@ -508,7 +508,7 @@ CharSettings['vault_info'] = {} unless CharSettings['vault_info'].class == Hash
 
 kronar_banks = ['Crossings', 'Dirge', 'Leth Deriel', 'Ilaya Taipa']
 lirum_banks = ["Aesry Surlaenis'a", "Hara'jaal", "Mer'Kresh", "Muspar'i", 'Ratha', 'Riverhaven', "Rossman's Landing", 'Therenborough', 'Throne City']
-dokora_banks = ['Ain Ghazal', 'Boar Clan', "Chyolvea Tayeu'a", 'Hibarnhvidar', 'Shard']
+dokora_banks = ['Ain Ghazal', 'Boar Clan', "Chyolvea Tayeu'a", 'Hibarnhvidar', 'Shard', "Raven's Point"]
```
Comment (rpherbig) [inline drinfomon.lic]: Does `Inner Hibarnhvidar` need to be added to this list?

File: drinfomon.lic [hunk 2]
```diff
@@ -518,12 +518,14 @@ bank_titles = {
   'Crossings'					=> ['[[Provincial Bank, Teller]]'],
   'Dirge'						  => ["[[Dirge, Traveller's Bank]]"],
   "Hara'jaal"					=> ['Unknown Description'],
-  'Hibarnhvidar'			=> ['[[Second Provincial Bank of Hibarnhvidar, Teller]]'],
+  'Hibarnhvidar'			=> ['[[Second Provincial Bank of Hibarnhvidar, Teller]]', "[[Hibarnhvidar, Teller Windows]]"],
   'Ilaya Taipa'				=> ['Unknown Description'],
+  'Inner Hibarnhvidar'				=> ["[Hibarnhvidar, Teller Windows]"],
```
Comment (rpherbig) [inline drinfomon.lic]: There's only one set of square brackets in the room name, not two?

File: drinfomon.lic [hunk 3]
```diff
@@ -518,12 +518,14 @@ bank_titles = {
...
   'Leth Deriel'				=> ['[[Imperial Depository, Domestic Branch]]'],
   "Mer'Kresh"					=> ["[[Harti Clemois Bank, Teller's Window]]"],
   "Muspar'i"					=> ["[[Old Lata'arna Keep, Teller Windows]]"],
   'Ratha'						  => ['[[Lower Bank of Ratha, Cashier]]', '[[Sshoi-sson Palace, Grand Provincial Bank, Bursarium]]'],
+  "Raven's Point"				=> ["[Bank of Raven's Point, Depository]"],
```
Comment (rpherbig) [inline drinfomon.lic]: There's only one set of square brackets in the room name, not two?

File: drinfomon.lic (hunk 1 above)
Comment (Sarvatt) [reply drinfomon.lic]: There isn't one for the 2 banks that share a balance on Ratha so I assumed not

**Comment by Sarvatt** [comment]
Thanks for that, I pushed the wrong branch apparently but that should be all fixed up on this branch now.

**Pull request merged**